### PR TITLE
Update to rails 5.0.0

### PR DESCRIPTION
**Pull request by Paxa**

### CODE REVIEW

**Comment by Paxa** [comment]
I got this:
```
rake appraisal:rails50 test_jdbc_mysql
```
```
NoMethodError: undefined method `accept' for nil:NilClass
                                                                                        to_sql at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/connection_adapters/abstract/database_statements.rb:12
                                                                                    select_all at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/connection_adapters/abstract/database_statements.rb:32
                                                                                    select_all at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/connection_adapters/abstract/query_cache.rb:70
                                                                                   find_by_sql at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/querying.rb:39
                                                                                  exec_queries at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/relation.rb:699
                                                                                          load at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/relation.rb:580
                                                                                       records at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/relation.rb:260
                                                                                     find_take at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/relation/finder_methods.rb:525
                                                                                          take at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/relation/finder_methods.rb:100
                                                                                       find_by at /Users/pavel/.rbenv/versions/jruby-9.1.2.0/lib/ruby/gems/shared/gems/activerecord-5.0.0/lib/active_record/relation/finder_methods.rb:78
```

**Comment by Paxa** [comment]
Solved `undefined method 'accept' for nil:NilClass`, problem was with recent arel gem

File: lib/arjdbc/jdbc/arel_support.rb [hunk 1]
```diff
@@ -1,3 +1,31 @@
+if Arel::VERSION >= "7.1.0"
```
Comment (kares) [inline lib/arjdbc/jdbc/arel_support.rb]: not sure about the right place to do the patch - maybe place these where visitors live (also make sure arel is loaded).
new features are usually adapter to work backwards although here (quick look a arel's commits) the patch might do.

**Comment by kares** [comment]
thanks for the PR - not sure how to proceed ... yet. the **rails-5** branch is off **1-3-stable** but has breaking changes in terms of 1.3 compatibility with JRuby as well as AR versions. this seems to bring those changes further down the path. thus I rather hold off merging until a solid Rails 5 release plan is in place.

anyways I much appreciate your effort - thanks!

File: lib/arjdbc/jdbc/arel_support.rb (hunk 1 above)
Comment (Paxa) [inline lib/arjdbc/jdbc/arel_support.rb]: It was removed here as unused https://github.com/rails/arel/commit/62e3596db89570b9d48b13c9966d754d5859d0dc

I didn't find any alternative yet

**Comment by zoras** [comment]
@kares @headius Any chance of merging the PR? I would really love to migrate to Rails 5 with pg support in jruby. Let me know how I can help.

**Comment by kares** [comment]
@zoras unfortunately, as far as I know NO adapter (except SQLite3) is Rails 5 complete/tested.
PR is not bad but "hacky" ... if arel removed those AR-JDBC's code should deal with that without forcing things back onto the `Arel` name-space (as my original comment above suggested). 

code-base is hard to manage as is and there's little :purple_heart: coming back ... these days.

will look into an alternative piece of code, although not sure I like the rails-5 branch as is ... maybe later

**Pull request closed without merging**